data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -1018,314 +577,11 @@ public URL getURL(String columnName) throws SQLException {
-
-    }
-
-    @Override
-    public void updateCharacterStream(int columnIndex, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateAsciiStream(String columnName, InputStream inputStream, long l) {
-
-    }
-
-    @Override
-    public void updateBinaryStream(String columnName, InputStream inputStream, long l) {
-
-    }
-
-    @Override
-    public void updateCharacterStream(String columnName, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateBlob(int columnIndex, InputStream inputStream, long l) {
-
-    }
-
-    @Override
-    public void updateBlob(String columnName, InputStream inputStream, long l) {
-
-    }
-
-    @Override
-    public void updateClob(int columnIndex, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateClob(String columnName, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateNClob(int columnIndex, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateNClob(String columnName, Reader reader, long l) {
-
-    }
-
-    @Override
-    public void updateNCharacterStream(int columnIndex, Reader reader) {
-
-    }
-
-    @Override
-    public void updateNCharacterStream(String columnName, Reader reader) {
-
-    }
-
-    @Override
-    public void updateAsciiStream(int columnIndex, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateBinaryStream(int columnIndex, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateCharacterStream(int columnIndex, Reader reader) {
-
-    }
-
-    @Override
-    public void updateAsciiStream(String columnName, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateBinaryStream(String columnName, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateCharacterStream(String columnName, Reader reader) {
-
-    }
-
-    @Override
-    public void updateBlob(int columnIndex, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateBlob(String columnName, InputStream inputStream) {
-
-    }
-
-    @Override
-    public void updateClob(int columnIndex, Reader reader) {
-
-    }
-
-    @Override
-    public void updateClob(String columnName, Reader reader) {
-
-    }
-
-    @Override
-    public void updateNClob(int columnIndex, Reader reader) {
-
-    }
-
-    @Override
-    public void updateNClob(String columnName, Reader reader) {
-
-    }
-
-    @Override
-    public <T> T getObject(int columnIndex, Class<T> aClass) {
-        return null;
-    }
-
-    @Override
-    public <T> T getObject(String columnName, Class<T> aClass) {
-        return null;
-    }
-
-    @Override
-    public <T> T unwrap(Class<T> aClass) {
-        return null;
-    }
-
-    @Override
-    public boolean isWrapperFor(Class<?> aClass) {
-        return false;
-    }
-
     public int count() {
         return rows.size();
     }
+
+    public boolean isNullAt(int columnIndex){
```

Comment:
IsColumnNull or simply IsNull

Reply:
took this from Spark - i think its more clear

Reply:
will take isNull